components/infobox/wikis/fighters/infobox_league_custom.lua

```diff
@@ -0,0 +1,196 @@
+		if _args.localcurrency and _args.localcurrency:lower() ~= 'usd' then
+			_args.prizepool = prizeMoney
+		else
+			_args.prizepoolusd = prizeMoney
+		end
+	end
+
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.getWikiCategories = CustomLeague.getWikiCategories
+
+	return league:createInfobox()
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	table.insert(widgets, Cell{name = 'Number of Players', content = {_args.player_number}})
+
+	return widgets
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'customcontent' then
+		if _args.circuit or _args.points or _args.circuit_next or _args.circuit_previous then
+			table.insert(widgets, Title{name = 'Circuit Information'})
+			table.insert(widgets, Cell{name = 'Circuit', content = {_args.circuit}})
+			table.insert(widgets, Cell{name = 'Circuit Tier', content = {
+				_args.circuittier and (_args.circuittier .. ' Tier') or nil}
+			})
+			table.insert(widgets, Cell{name = 'Tournament Region', content = {_args.region}})
+			table.insert(widgets, Cell{name = 'Points', content = {_args.points}})
+			table.insert(widgets, Chronology{content = {next = _args.circuit_next, previous = _args.circuit_previous}})
+		end
+
+	elseif id == 'prizepool' then
+		if _args.prizepoolassumed then
+			widgets[1].content[1] = Abbreviation.make(
+				widgets[1].content[1],
+				'This prize is assumed, and has not been confirmed'
+			)
+		end
+
+	elseif id == 'gamesettings' then
+		return {
+			Cell{name = 'Game', content = {_args.game}},
+			Cell{name = 'Version', content = {_args.version}},
+		}
+	end
+	return widgets
+end
+
+function CustomLeague:addToLpdb(lpdbData, args)
+	lpdbData.participantsnumber = string.gsub(_args.player_number or '', ',', '')
+
+	if Logic.readBool(_args.overview) then
+		lpdbData.game = 'none'
+	end
+
+	lpdbData.extradata.assumedprizepool = tostring(_args.prizepoolassumed)
+	lpdbData.extradata.circuit = _args.circuit
+	lpdbData.extradata.circuit_tier = _args.circuit_tier
+
+	return lpdbData
+end
+
+function CustomLeague:defineCustomPageVariables()
+	--Legacy vars
+	Variables.varDefine('tournament_tier', _args.liquipediatier or '')
+	Variables.varDefine('prizepoolusd', Variables.varDefault('tournament_prizepoolusd'))
+	Variables.varDefine('tournament_entrants', string.gsub(_args.player_number or '', ',', ''))
+	Variables.varDefine('tournament_region', _args.region)
+	Variables.varDefine('assumedpayout', tostring(_args.prizepoolassumed))
+	Variables.varDefine('localcurrency', Variables.varDefault('tournament_currency', ''):upper())
+	Variables.varDefine('circuittier', _args.circuittier)
+	Variables.varDefine('tournament_circuit', _args.circuit)
+	Variables.varDefine('seriesabbr', _args.abbreviation)
+	Variables.varDefine('circuitabbr', _args.circuitabbr)
```

Comment:
are all of them legacy?
from looking at them only `tournament_tier` and `prizepoolusd` seem legacy